src/sctools/platform.py

```diff
@@ -418,6 +420,52 @@ def merge_count_matrices(cls, args: Iterable[str]=None) -> int:
         count_matrix.save(args.output_stem)
 
         return 0
+    
+    def group_qc_outputs():
+        """Commandline entrypoint for parsing picard metrics files, hisat2 and rsem statistics log files.
+        Parameters
+        ----------
+        args:
+            file_names: array of files
+            output_name: prefix of output file name.
+            metrics_type: Picard, PicardTable, HISAT2, RSEM and Core.
+        Returns
+        ----------
+        return: 0 
+            return if the program completes successfully.
+        """
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+                "-f",
+                "--file_names",
+                dest="file_names",
+                nargs='+',
+                required=True,
+                help="a list of files to be parsed out.")
+        parser.add_argument(
+                "-o",
+                "--output_name",
+                dest="output_name",
+                required=True,
+                help="The output file name")
+        parser.add_argument(
+                "-t",
+                "--metrics_type",
+                dest="metrics_type",
+                required=True,
+                help="a list of string to represent metrics types,such Picard, PicardTable, HISAT2,RSEM, Core")
```

Comment:
You can use the `options` parameter here to define valid values.

Reply:
add `choices`

Reply:
There are some whitespace inconsistencies in the help string. Also, this isn't a list, is it? It's a single string?